catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -111,7 +111,7 @@ define([
                     return;
                 }
             } else {
-                if (this.isEmpty() && !this.get('regex') && !this.get('required')) {
+                if (!this.get('value') && !this.get('regex') && !this.get('required')) {
```

Comment:
I just completed some more thorough testing to make sure that this change does not affect the other types of fields.